src/components/SettingsDialog/Components/SettingsAccordion.scss

```diff
@@ -0,0 +1,47 @@
+@import "src/constants/style";
+
+.accordion-item {
+  width: 100%;
+
+  &__header {
+    width: 100%;
+    height: 48px;
+    font-size: 14px;
+    display: flex;
+    padding: $padding--small $padding--medium;
+    background-color: unset;
+    border: none;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__header-icon {
+    height: 24px;
+    color: var(--accent-color);
+    transform: rotate(90deg);
+
+    &--open {
+      transform: rotate(-90deg);
```

Comment:
kriegen wir da noch eine Animation rein? Vielleicht sogar für die carousels, dass die nach unten sliden oder so?

Reply:
Hab nochmal n bisschen rumprobiert aber sieht mMn alles 💩 aus. Wenn du da was gutes findest, be my guest. Fand ne Animation nämlich auch schön aber die Animationen waren nicht schön :(
https://stackoverflow.com/questions/3508605/how-can-i-transition-height-0-to-height-auto-using-css
Falls du Dinge ausprobieren willst

Reply:
Könnte man sich wann anders mal anschauen. Lege evtl. nachher ein neues Issue an